src/common/entryPointHandler.ts

```diff
@@ -11,34 +16,37 @@ export class EntryPointHandler {
     private outputChannel: OutputChannel;
 
     constructor(outputChannel?: OutputChannel) {
```

Comment:
With this change we only use the outputChannel to create the isDebugeeProcess  flag.
We could pass that to the constructor instead, and don't require the outputChannel any more.